speed-dreams/source-2.2.3/src/simulatedDrivingAssistance/backend/Mediator.h

```diff
@@ -15,24 +15,22 @@ class Mediator
 {
 public:
 	void DriveTick(tCarElt* p_car, tSituation* p_situation);
-	void RaceStart(tTrack* p_track, void* p_carHandle, void** p_carParmHandle, tSituation* p_situation);
+	void RaceStart(tTrack* p_track, void* p_carHandle, void** p_carParmHandle, tSituation* p_situation, bool p_recordBB);
 	void RaceStop();
 
 	void SetTask(Task p_task);
-
     void SetIndicatorSettings(tIndicator p_indicators);
-
     void SetInterventionType(InterventionType p_type);
+    void SetPControlSettings(tParticipantControl p_pControl);
 
     void SetMaxTime(int p_maxTime);
-
     void SetUserId(char* p_userId);
 
     void SetDataCollectionSettings(tDataToStore p_dataSetting);
 
-    tIndicator GetIndicatorSettings();
-
-    InterventionType GetInterventionType();
+    tIndicator          GetIndicatorSettings();
+    InterventionType    GetInterventionType();
+    tParticipantControl GetPControlSettings();
```

Comment:
Don't know if this is needed if Mediator handles this itself internally (see other comment)

Reply:
It is also used in rthumandriver